src/types/passport.d.ts

```
@@ -0,0 +1,17 @@
+declare module 'passport-kakao' {
+  import { Strategy } from 'passport';
+
+  export class Strategy extends Strategy {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Strategy'. Consider to delete it or rename it.

'Strategy' is defined here:


(lint/suspicious/noRedeclare)